validator/lib/validateTileset.js

```diff
@@ -36,6 +36,12 @@ function validateTopLevel(tileset) {
         return 'Tileset must declare its geometricError as a top-level property.';
     }
 
+    if (defined(tileset.root.refine)) {
+        if (tileset.root.refine !== 'ADD' && tileset.root.refine !== 'REPLACE') {
+            return 'Refine property in root tileset must have either ADD or REPLACE as its value';
+        }
+    }  
```

Comment:
This area needs to return an error message if `refine` is not defined for the root.

Also the check that `refine` is `ADD` or `REPLACE` is handled below and doesn't need to be included here.